question/bank/managecategories/templates/categories.mustache

```diff
@@ -0,0 +1,90 @@
+{{!
+    This file is part of Moodle - http://moodle.org/
+
+    Moodle is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Moodle is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+}}
+{{!
+    @template qbank_managecategories/categories
+
+    This template renders the base for category view.
+
+    Example context (json):
+    {
+        "helpstringhead": "<h2> Edit categories <a href='someurl'>...</a></h2>",
```

Comment:
the mustache is not parsed properly, check /admin/tool/templatelibrary/index.php for more info, mainly I guess the wrong use of "" and '' in some places

Reply:
Done.